charts/opentelemetry-demo/values.yaml

```diff
@@ -519,9 +519,17 @@ components:
     env:
       - name: KAFKA_ADVERTISED_LISTENERS
         value: 'PLAINTEXT://{{ include "otel-demo.name" . }}-kafka:9092'
+      - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+        value: 'http://{{ include "otel-demo.name" . }}-otelcol:4317'
+      - name: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
+        value: 'http://{{ include "otel-demo.name" . }}-otelcol:4317'
```

Comment:
does this need `/v1/metrics`?

Reply:
It works without the path suffix. Though I now see the inconsistency across the demo components of using the standard vs telemetry-specific endpoints with and without paths.